airflow/providers/google/cloud/operators/datafusion.py

```diff
@@ -34,21 +34,12 @@
 from airflow.providers.google.cloud.operators.cloud_base import GoogleCloudBaseOperator
 from airflow.providers.google.cloud.triggers.datafusion import DataFusionStartPipelineTrigger
 from airflow.providers.google.cloud.utils.datafusion import DataFusionPipelineType
+from airflow.providers.google.cloud.utils.helpers import resource_path_to_dict
 
 if TYPE_CHECKING:
     from airflow.utils.context import Context
 
 
-class DataFusionPipelineLinkHelper:
```

Comment:
is this class part of the public API?
Do we need to deprecate it first?

Reply:
You are right. Updated PR.